iothub/service/src/Registry/Models/JobProperties.cs

```diff
@@ -20,14 +20,61 @@ public class JobProperties
             JobStatus.Cancelled,
         };
 
+        /// <summary>
+        /// Serialization constructor
+        /// </summary>
+        internal JobProperties() { }
+
+        /// <summary>
+        /// Creates an instance of this class for the import job.
+        /// </summary>
+        /// <remarks>
+        /// Other relevant, optional properties to set include:
+        /// <list type="bullet">
+        /// <item>InputBlobName, to override the default.</item>
+        /// <item>StorageAuthenticationType, if using identity-based instead of SAS in InputBlobContainerUri.</item>
+        /// <item>ManagedIdentity, if the IoT hub is not configured with a managed identity and the service app will specify
+        /// which identity has permissions to the storage account.</item>
+        /// <item>IncludeConfigurations, if the import job should also include configurations.</item>
+        /// <item>ConfigurationsBlobName, if the import job includes configurations and to override the default blob name.</item>
+        /// </list>
+        /// </remarks>
+        public JobProperties(Uri inputBlobContainerUri, Uri outputBlobContainerUri)
```

Comment:
Instead of factory methods, we have 2 ctors, one for import and one for export. Comments indicate which are for which, as well as what additional properties are relevant, but optional.